scripts/test/run-tests.sh

```diff
@@ -29,7 +29,7 @@ RASPBIAN_IMAGE_URL="http://downloads.raspberrypi.org/raspbian_lite/images/raspbi
 UBUNTU_IMAGE_URL="https://downloads.mender.io/mender-convert/images/Ubuntu-Bionic-x86-64.img.gz"
 
 ## Auto-update
-UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/20.04.2/release/ubuntu-20.04.2-preinstalled-server-armhf+raspi.img.xz"
+UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/20.04/release/ubuntu-20.04.2-preinstalled-server-armhf+raspi.img.xz"
```

Comment:
I probably don't understand the bot, but won't this be overwritten by the next update? Or does it use the existing link structure as input?

Reply:
Not wholly sure what you mean by next update, but if you mean the next run of the bot, then yes, in reality it overwrites it every-time, but this is not really shown, since in this case, the link is the same.
I guess we could add a check for this, but it's not really necessary.

Reply:
Ah, so this link is actually generated?

Reply:
Yes